Source/NSTableView.m

```diff
@@ -6674,39 +6716,46 @@ - (id) _objectValueForTableColumn: (NSTableColumn *)tb
 			      row: (NSInteger) index
 {
   id result = nil;
-  GSKeyValueBinding *theBinding;
-
-  theBinding = [GSKeyValueBinding getBinding: NSValueBinding 
-                                   forObject: tb];
-  if (theBinding != nil)
-    {
-      return [(NSArray *)[theBinding destinationValue]
-                 objectAtIndex: index];
-    }
-  else if ([_dataSource respondsToSelector:
-		    @selector(tableView:objectValueForTableColumn:row:)])
+  
+  if (_viewBased == NO)
     {
-      result = [_dataSource tableView: self
-			    objectValueForTableColumn: tb
-			    row: index];
+      GSKeyValueBinding *theBinding;
+      
+      theBinding = [GSKeyValueBinding getBinding: NSValueBinding 
+				       forObject: tb];
+      if (theBinding != nil)
+	{
+	  return [(NSArray *)[theBinding destinationValue]
+		     objectAtIndex: index];
+	}
+      else if ([_dataSource respondsToSelector:
+			   @selector(tableView:objectValueForTableColumn:row:)])
+	{
+	  result = [_dataSource tableView: self
+				objectValueForTableColumn: tb
+				      row: index];
+	}
     }
-
+      
   return result;
```

Comment:
Don't view based table views have any value?

Reply:
Actually it does... working on this change.

Reply:
fixed